java/src/main/java/info/technicallycorrect/qi/ArraysAndStrings.java

```diff
@@ -2,12 +2,20 @@
 
 public class ArraysAndStrings {
 
-  public static boolean uniqueChars(final String input) {
-    return false;
-  }
+    public static boolean uniqueChars(final String input) {
+        //Will work only if string is ASCII
+        boolean[] bool_char = new boolean[256];
+        for (int i = 0; i < input.length(); i++) {
+            int val = input.charAt(i);
+            if (bool_char[val])
```

Comment:
It's better to use brackets here.  There is a famous case where Apple made a massive security error because they didn't.

http://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/

Also, this case the `if-then` cause is symmetric.  Ideally, the code should reflect that!